src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Web/appsettings.json

```diff
@@ -36,6 +36,44 @@
         }
       }
     ]
-  }
+  },
   //#endif
+  "OrchardCore": {
```

Comment:
I think this should be reverted since it's now part of the code as defaults.

Reply:
I think it's useful if we run the source, while the defaults are useful if we run out of the source

Correct me if I'm wrong?

Reply:
We always run partially from the source, the application itself ;) What @sebastienros says is that it is not necessary when building an application by using the `Templates` so that we don't have to maintain 2 `appsettings.json`, and then people can still update it as they want.

Reply:
We changed this file when we added the cultures to the appsettings. Then we removed the changes in appsettings so we should also remove it there.

Reply:
Oups i missed that it has also been removed under the web app ;)

Reply:
Just checked, `SupportedCultures` are still defined in the appsettings of the web app.